core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -472,12 +512,12 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
         }}
         {if (hasInput) {
           <td sorttable_customkey={inputSortable}>
-            {inputReadable}
+            {s"$inputReadable / $inputRecords"}
```

Comment:
good point. I could add checks in the UI code for inputRecords < 0 and now show it. You think thats worth doing or leaving it as -8?